blackmarket/UI/Common/PaymentMethodsView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  PaymentMethodsView.swift
+//  blackmarket
+//
+//  Created by Daniel Parra on 12/6/22.
+//
+
+import SwiftUI
+
+struct PaymentMethodsView: View {
+    var body: some View {
+        VStack(spacing: UI.PaymentMethodsView.VStack.spacing) {
+            Text("Payment methods")
+                .bold()
+                .font(.headline)
+            HStack(spacing: UI.PaymentMethodsView.HStack.spacing) {
+                VStack {
+                    Image("cardIcon")
+                    Text(LocalizedString.PaymentMethodsScreen.paymentMethodsCreditTitle)
+                }
+                Divider()
+                    .frame(
+                        width: UI.PaymentMethodsView.Divider.width,
```

Comment:
do you need to set the width = 1.0 here? According to the docs, a Divider will be laid out vertically if used within a HStack, or you can simply set the height. Check below as well.